include/class.task.php

```diff
@@ -1062,6 +1062,13 @@ function getVar($tag) {
             break;
         case 'last_update':
             return new FormattedDate($this->last_update);
+        case 'description':
+            return Format::display($this->getThread()->getEntries()->filter(array(
+                'type' => 'M',
+                'pid' => 0,
+            ))->first()) ?: '';
```

Comment:
I think instead of:
```
return Format::display($this->getThread()->getEntries()->filter(array(
    'type' => 'M',
    'pid' => 0,
))->first()) ?: '';
```
I can set the `FLAG_ORIGINAL_MESSAGE` flag when adding the Task Description (if provided) and then select the entry with the flag set like:
```
$entry = $this->getThread()->getEntries()->filter(array(
    'type' => 'M',
    'flags__hasbit' => ThreadEntry::FLAG_ORIGINAL_MESSAGE,
    ))
    ->first();
return $entry ? Format::display($entry->getBody()) : '';
break;
```

I think this will be better than filtering all threads by `type` and `pid`. Any thoughts?